src/services/BuildingService.ts

```diff
@@ -1,68 +1,66 @@
-import axios from 'axios';
+import { typedRequest } from '../../src/services/api/typedRequest';
+import type { paths, components } from '../../src/services/api/platform-schema';
 
-export interface BuildingUnit {
-  id?: string;
-  title: string;
-  description?: string;
-  location?: string;
-  livingSpace?: number;
-  commercialSpace?: number;
-  usableSpace?: number;
-  heatingSpace?: number;
-  address?: {
-    street: string
-    city: string
-    province: string
-    zip: string
-    country: string
-  }
-}
+// OpenAPI schema types
+export type CreateBuildingRequest =
+  paths['/api/v1/projects/{projectId}/properties/{propertyId}/buildings']['post']['requestBody']['content']['application/json'];
 
-class BuildingService {
-  private readonly baseUrl: string = '/api/v1/projects';
+// There is no GET success response, so we fallback to BuildingJson directly wait until backend update 
```

Comment:
The GET has been implemented with PR #580 

See https://github.com/remsfal/remsfal-frontend/blob/02687b1cd8c8f93dda96bdf16d6cc14db0dc4674/src/services/api/platform-schema.ts#L689